modules/rest-api/pages/rest-bucket-create.adoc

```diff
@@ -96,11 +98,7 @@ Parameters in the _General_ group include:
 * Parameters that _can_ be specified on bucket creation, but if not specified, acquire a default value.
 They include:
 
-** Parameters that _can_ be edited after bucket creation; these being xref:rest-api:rest-bucket-create.adoc#evictionpolicy[evictionPolicy], xref:rest-api:rest-bucket-create.adoc#durabilityminlevel[durabilityMinLevel], xref:rest-api:rest-bucket-create.adoc#threadsnumber[threadsNumber], xref:rest-api:rest-bucket-create.adoc#replicanumber[replicaNumber], xref:rest-api:rest-bucket-create.adoc#compressionmode[compressionMode], xref:rest-api:rest-bucket-create.adoc#maxttl[maxTTL], xref:rest-api:rest-bucket-create.adoc#flushenabled[flushEnabled], xref:rest-api:rest-bucket-create.adoc#magmaseqtreedatablocksize[magmaSeqTreeDataBlockSize],
```

Comment:
Just curious - any particular reason to re-order these parameter here?

Reply:
I was trying to figure out which of the parameters that can be changed after bucket creation matched the ones you mention could be changed during a migration. To make that easier, I rearranged them in alphabetical order in the doc. However, someone else made a change to the same part of the doc, resulting in a hairy merge conflict. I just reverted my changes and added back the backend storage type parameter.